_pages/media.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Media
+description: |
+  As the language and the community continues to grow it is important to have
+  basic understanding about how and for what purposes the Crystal brand and
+  associated works can be used.
+---
+
+## Assets
+
+Thanks for showing interest in Crystal! As the language and the community
+continues to grow we think it is important to have basic rules of thumb about
+how and for what purposes you can use the Crystal brand.
+
+We’ve put together some assets and guidelines to help you determine the best way
+to use our brand and logos.
+
+### Logo
+
+<div class="cards-list">
```

Comment:
Ensure people follow the guidelines, something of the form:

```suggestion

> **NOTE** Important
> Make sure to follow the [guidelines](???) described below when using any of this material.

<div class="cards-list">
```

Reply:
```suggestion

> **NOTE** Important
> Make sure to follow the [guidelines](#guidelines) described below when using any of this material.

<div class="cards-list">
```

Reply:
The callout is coming weird (everything in one line), despite of following the rules described in the `style-guide`
<img width="1011" alt="image" src="https://github.com/crystal-lang/crystal-website/assets/5183634/95cf297c-06f9-431e-b849-2dbaf065739e">

Reply:
it's missing the `:`, that might be it?